bucket/trafficmonitor.json

```diff
@@ -14,7 +16,27 @@
         }
     },
     "extract_dir": "TrafficMonitor",
-    "bin": "TrafficMonitor.exe",
+    "installer": {
+        "script": [
+            "if (!(Test-Path \"$persist_dir\\config.ini\")) { Set-Content \"$dir\\config.ini\" @('[general]', 'check_update_when_start = false') }",
+            "Set-Content \"$dir\\global_cfg.ini\" @('[config]', 'portable_mode = true')"
```

Comment:
Always use `-Encoding Ascii` for *-Content cmdlets

Reply:
Why global is not persisted?

Why user cannot change it? If you always configure it statically users changes will be always overriden.

Reply:
The global_cfg.ini file has only one setting and it's what's being set in the manifest. If one doesn't want the portable version why even bother installing it with scoop?

Reply:
Not your decision to make.